lib/xclarity_client/endpoints/compliance_policy.rb

```diff
@@ -0,0 +1,19 @@
+module XClarityClient
+  class CompliancePolicy < Endpoints::XclarityEndpoint
+
+    BASE_URI = '/compliancePolicies'.freeze
+    LIST_NAME = 'updatePolicyList'.freeze
+    SUB_URIS = {
+                 applicableFirmware: "#{BASE_URI}/applicableFirmware",
+                 persistedResult: "#{BASE_URI}/persistedResult",
+                 compareResult: "#{BASE_URI}/compareResult"
+               }
```

Comment:
in this case, you can indent at the same level of `SUB_URIS` constant

Reply:
updated it